tox.ini

```diff
@@ -12,7 +12,7 @@ usedevelop = True
 commands =
     pytest tests --verbose --cov --cov-report term-missing --junitxml=test_report.xml --doctest-modules src
     pytest --pylint src/
-    pytest --pylint tests/
+    pytest --pylint tests/ --pylint-rcfile=tests/.pylintrc
```

Comment:
Does pylint not read the rc file automatically?

Reply:
In a project context, only from a `pylintrc` or `.pylintrc` at the project root, otherwise the location needs to be specified with `--pylint-rcfile`.